sda/cmd/auth/frontend/templates/oidc.html

```diff
@@ -30,27 +30,20 @@
             <p class="lead text-center">
               Welcome, {{.User}}!
             </p>
-            {{if .Passport}}
+            {{if .ResignedToken}}
```

Comment:
Putting the comment here as the line above that needs fixing can't be commented on.

the `Welcome, {{.User}}!` hsould be `Welcome, {{.Profile}}!` in order to display  the users full name instead of the username.

Reply:
Fixed in 476555bf